aiven/client/cli.py

```diff
@@ -1343,6 +1343,7 @@ def _get_store_from_args(self) -> Store:
             store = Store.skip
         return store
 
+    # This method should be deprecated and removed over time. Check service__connection_info__kcat
```

Comment:
I would rather use `warnings.warn("Use the kcat method instead", DeprecationWarning)`

Reply:
thanks!